src/ControllerV2/DocumentController.php

```diff
@@ -203,7 +200,7 @@ public function toggleVisibility(Request $request, Document $document, DocumentM
     public function moveToFolder(
         Request $request,
         Document $document,
-        DocumentManager $manager,
+        FolderableEntityManager $manager,
```

Comment:
Not sure Entity is a good name, especially if the service name ends with EntityManager :)